### DISCUSSION TITLE
Adapters in ClientVoiceManager usage.

### DISCUSSION THREAD

**Discussion by its-mrarsikk** [discordjs/discord.js#7843] [Q&A] [answered]
Hello! I am writing a music bot (_but it plays only rickroll_) on JS, and i am stuck on getting the current voice channel of an user. This is my current call.
```js
!message.author.client.voice.channel
```
If this condition is true, it will send me a message with these contents: `Join a channel first!`. But, what can you expect, it sends it without any care whether am i in a voice channel or not. `m!play` is the command.

Versions:
```js
Node: v17.3.1
Discord.JS: 13.6.0 (as i found in package.json of node_modules/discord.js)
Discord directory (in node_modules): 0.8.2
Distube (package.json): 3.3.4
```

_**Message proof**_
![image](https://user-images.githubusercontent.com/94739102/164974410-029d9302-9d4d-4a24-9f49-2130f1317062.png)
_**Voice proof**_
![image](https://user-images.githubusercontent.com/94739102/164974466-b5efea4e-69ca-4407-85c2-8a60dd995bb5.png)

What to do?

_Originally posted by @its-mrarsikk in https://github.com/discordjs/discord.js/discussions/7834_

**Comment by Syjalo** [comment]
**The issue tracker is only for bug reports and enhancement suggestions. If you have a question, please ask it in the [Discord server](https://discord.gg/djs) instead of opening an issue – you will get redirected there anyway.**

**Comment by monbrey** [comment, marked as answer]
`message.author.client` is a reference to *your* Client object. What you're accessing is your bots voice state, not theirs.

What you're probably looking for is `message.member.voice`

**Reply by its-mrarsikk** [reply]
Thanks. working.
